### PR TITLE
SIDE-1004 - Add template docs back

### PR DESCRIPTION
**Pull request by dmorrison-olive**
Addressing @richseviora's feedback [here](https://github.com/open-olive/loop-development-kit/pull/15#discussion_r523056832).

The `@template` tag [seems to be supported](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template) for TypeScript/JSDoc.

### CODE REVIEW

File: ldk/node/src/hostClients/stoppables.ts [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
  * The StreamListener type is a function provided to start a stream. It accepts two parameters and returns nothing.
  *
+ * @template T - The type of the messages sent.
  * @param error - Null, unless an error is present.
  * @param input - The event that's being generated.
  */
 export type StreamListener<T> = (error: string | null, input?: T) => void;
 
+/**
+ * @template T - The type of the messages sent.
```
Comment (richseviora) [inline ldk/node/src/hostClients/stoppables.ts:11]: Sorry, these should be `@typeParam` tags not `@template`.

**Review by richseviora — CHANGES_REQUESTED** [review]
Almost there :)

File: ldk/node/src/hostClients/stoppables.ts (hunk 1 above)
Comment (dmorrison-olive) [reply ldk/node/src/hostClients/stoppables.ts:11]: It seems to be an alias? See [here](https://typedoc.org/guides/doccomments/#typeparam-param-name) and the comment "You may also use the template tag".

`@typeParam` gives me an ESLint warning, whereas `@template` doesn't. I don't see where I can configure this either. 😕

Comment (richseviora) [reply ldk/node/src/hostClients/stoppables.ts:11]: OH! TIL!

That `@template` is an alias for `@typeParam` is totally not intuitive at all but I'll take it!

I think you can configure whichever rule is complaining about to support additional tags in the eslint config file

**Review by richseviora — APPROVED** [review]

**Review by BillyMurrayOlive — APPROVED** [review]

**Pull request merged**